src/gso.ts

```diff
@@ -277,4 +277,44 @@ export class GSO {
       },
     );
   }
+
+  /**
+   * Create an instruction for withdraw
+   */
+  public async createWithdrawInstruction(
+    projectName: string,
+    baseMint: PublicKey,
```

Comment:
Should this be inferred from base account or vice versa? Similar to xBaseMint

Reply:
The user can declare any account, not just the ATA.
We could lookup the token account on chain, but we should have the base mint available. It is redundant, but shouldnt be harmful.

We should probably clean up the need for baseMint later, but will be easier if we store it on the GSOState. A future optimization